File: javascript/searching/linear-search.js
function linearSearch(arr, searchElement) {
  let isFound = false;
  arr.forEach(element => {
    if (element === searchElement) {
      isFound = true;
    }
  });
  return isFound;
}

const arr = [10, 20, 80, 30, 60, 50, 110, 100, 130, 170];
const searchElement = 0;

linearSearch(arr, searchElement); //?

File: python/searching/linear-search.py
def linear_search(arr, n, x):
  for i in range(n):
    if arr[i] == x:
      return True
  return False

arr = [23,512,214,12,5,67,1,4,65]
result = linear_search(arr, len(arr), 214)
print('Search Element is found', result)
